nncf/torch/dynamic_graph/io_handling.py

```diff
@@ -227,7 +229,7 @@ def __init__(self, forward_args: Tuple, forward_kwargs: Dict):
         self._forward_args = forward_args
         self._forward_kwargs = forward_kwargs
```

Comment:
This is the `__init__` that should be on the `ExactInputsInfo` level of the hierarchy.

Reply:
done